pkgs/by-name/ju/junest/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, wget }:
+
+stdenv.mkDerivation rec {
+  version = "7.4.8";
+  pname = "junest";
+  src = fetchFromGitHub {
+    owner = "fsquillace";
+    repo = "junest";
+    rev = "refs/tags/${version}";
+    hash = "sha256-9yrQ721fHUxXEZ0mh27SB8yoUH67ltOktUq3kr4qrBc=";
+  };
+
+  dontBuild = true;
+
+  buildInputs = [
+    wget
```

Comment:
wget is not a buildInput, it's a runtime dependency 

if you want to provide wget as runtime dependency you can use https://nixos.org/manual/nixpkgs/unstable/#fun-wrapProgram

but it's not strictly required, I'll leave it up to you

Reply:
Done!